src/test/datascience/variableView/variableView.vscode.test.ts

```diff
@@ -0,0 +1,103 @@
+'use strict';
+import { assert } from 'chai';
+import * as sinon from 'sinon';
+import { ICommandManager, IVSCodeNotebook } from '../../../client/common/application/types';
+import { IDisposable } from '../../../client/common/types';
+import { Commands, VSCodeNotebookProvider } from '../../../client/datascience/constants';
+import { IVariableViewProvider } from '../../../client/datascience/variablesView/types';
+import { IExtensionTestApi } from '../../common';
+import { initialize } from '../../initialize';
+import {
+    canRunNotebookTests,
+    closeNotebooks,
+    closeNotebooksAndCleanUpAfterTests,
+    deleteAllCellsAndWait,
+    executeCell,
+    insertCodeCell,
+    startJupyter,
+    trustAllNotebooks,
+    waitForExecutionCompletedSuccessfully,
+    waitForKernelToGetAutoSelected
+} from '../notebook/helper';
+import { INotebookEditorProvider } from '../../../client/datascience/types';
+import { OnMessageListener } from '../vscodeTestHelpers';
+import { InteractiveWindowMessages } from '../../../client/datascience/interactive-common/interactiveWindowTypes';
+import { verifyViewVariables } from './variableViewHelpers';
+
+suite('DataScience - VariableView', () => {
+    let api: IExtensionTestApi;
+    const disposables: IDisposable[] = [];
+    let commandManager: ICommandManager;
+    let variableViewProvider: IVariableViewProvider;
+    let editorProvider: INotebookEditorProvider;
+    let vscodeNotebook: IVSCodeNotebook;
+    suiteSetup(async function () {
+        this.timeout(120_000); // IANHU: From other tests? Reduce this?
+        api = await initialize();
+
+        // Don't run if we can't use the native notebook interface
+        if (!(await canRunNotebookTests())) {
+            return this.skip();
+        }
+        await trustAllNotebooks();
+        await startJupyter(true);
+        sinon.restore();
+        commandManager = api.serviceContainer.get<ICommandManager>(ICommandManager);
+        variableViewProvider = api.serviceContainer.get<IVariableViewProvider>(IVariableViewProvider);
+        vscodeNotebook = api.serviceContainer.get<IVSCodeNotebook>(IVSCodeNotebook);
+        editorProvider = api.serviceContainer.get<INotebookEditorProvider>(VSCodeNotebookProvider);
+    });
+    setup(async function () {
+        sinon.restore();
+
+        // Create an editor to use for our tests
+        await editorProvider.createNew();
+        await waitForKernelToGetAutoSelected();
+        await deleteAllCellsAndWait();
+        assert.isOk(vscodeNotebook.activeNotebookEditor, 'No active notebook');
+    });
+    teardown(async function () {
+        await closeNotebooks(disposables);
+        await closeNotebooksAndCleanUpAfterTests(disposables);
+    });
+
+    // Cleanup after suite is finished
+    suiteTeardown(() => closeNotebooksAndCleanUpAfterTests(disposables));
+
+    // Test showing the basic variable view with a value or two
+    test('Can show VariableView', async function () {
```

Comment:
More tests in the queue. But wanted to get approval here and get one basic test running on CI before adding more.